spark/v4.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCallStatementParser.java

```diff
@@ -69,16 +70,9 @@ public static void stopSpark() {
   }
 
   @Test
-  public void testDelegateUnsupportedProcedure() {
-    assertThatThrownBy(() -> parser.parsePlan("CALL cat.d.t()"))
-        .isInstanceOf(ParseException.class)
-        .hasMessageContaining("Syntax error")
-        .satisfies(
-            exception -> {
-              ParseException parseException = (ParseException) exception;
-              assertThat(parseException.getErrorClass()).isEqualTo("PARSE_SYNTAX_ERROR");
-              assertThat(parseException.getMessageParameters().get("error")).isEqualTo("'CALL'");
-            });
+  public void testDelegateUnsupportedProcedure() throws ParseException {
+    LogicalPlan plan = parser.parsePlan("CALL cat.d.t()");
+    assertThat(plan.toString().contains("CALL cat.d.t()"));
```

Comment:
Since Spark 4.0 has Stored Procedure support, so this doesn't throw ParseException any more

Reply:
Shouldn't we just remove this test then? I'm not sure what we are checking here any more?